erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```diff
@@ -253,14 +254,20 @@
   {
    "fieldname": "column_break_19",
    "fieldtype": "Column Break"
+  },
+  {
+   "default": "1",
+   "fieldname": "post_change_gl_entries",
+   "fieldtype": "Check",
+   "label": "Post Ledger Entries for Given Change"
```

Comment:
@nextchamp-saqib Please add field description, no context for Change in Accounts Settings. The label can be "Create Ledger Entries for Change Amount"